assets/javascripts/kitten/components/form/field/components/input.js

```diff
@@ -1,34 +1,57 @@
 import React from 'react'
+import styled from 'styled-components'
+import classNames from 'classnames'
 import PropTypes from 'prop-types'
-import { Marger } from '../../../layout/marger'
 import { TextInput } from '../../../form/text-input'
 import { TextInputWithLimit } from '../../../form/text-input-with-limit'
 import { TextInputWithUnit } from '../../../form/text-input-with-unit'
 
-export const FieldInput = props => {
+const StyledFieldInput = styled.div`
+  &.k-FieldInput--noMargin {
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+`
+
+export const FieldInput = ({
+  limit,
+  unit,
+  noMargin,
+  className,
+  ...props
+}) => {
+
   let Input = TextInput
 
-  if (props.limit) {
+  if (limit) {
     Input = TextInputWithLimit
   }
 
-  if (props.unit) {
+  if (unit) {
     Input = TextInputWithUnit
   }
 
   return (
-    <Marger top="1">
+    <StyledFieldInput
+      className={classNames('k-FieldInput',
+        className, 
+        { 'k-FieldInput--noMargin': noMargin },
+        "k-u-margin-top-single",
+      )}
+    >
```

Comment:
Regarde si ça ne marcherait pas comme ça, sans toute la complication apportée par `styled components`

```suggestion
    <div
      className={classNames(
        className, 
        { 'k-u-margin-top-single': !noMargin },
      )}
    >
```